fixtures/packaging/requirejs/dev.html

```diff
@@ -1,15 +1,10 @@
 <html>
   <body>
     <script src="https://unpkg.com/requirejs@2.3.2/require.js"></script>
+    <script src="../../../build/dist/react.development.js"></script>
+    <script src="../../../build/dist/react-dom.development.js"></script>
     <div id="container"></div>
     <script>
-      requirejs.config({
```

Comment:
Is it worth still trying to test this? Does this trigger some other code path in RequireJS?

Maybe we could copy the fixtures?

Reply:
I wasn't sure which route you'd prefer.  I did test both before opening the PR.

Options:
1. Merge as-is.
2. Include `react-dom` as a script and `react` in this map (or vice versa - we could even have dev test one combo and prod test the other).
3. Make a new pair of fixtures that tests the map style.

Which would you prefer?  If you'd like both, would they each live as folders in `fixtures`, or as subfolders of `fixtures/requirejs`?

Reply:
Top-level folders in `fixtures` sound fine, let's keep both versions

Reply:
I've never used RequireJS before, so I'm certainly not the expert.  [Perusing its source](https://github.com/requirejs/requirejs/blob/master/require.js), it looks like `define(name` is inserted into `defQueue`, but it's not obvious how `config.paths` gets registered.

I didn't originally add 2 sets of fixtures because ensuring that `define(name` and `paths[name]` worked the same way seemed like a RequireJS concern (rather than a React one), but I'm happy to add a second set if you think it's worthwhile.  (I also don't know who/how fixtures are tested - is it just a manual smoketest, or is checking them part of a process?)

Reply:
Just a manual smoketest.

I'm not an expert either—do what you think is reasonable and we'll try to review :-)

Reply:
I'm inclined to leave it as-is.  My inclination is that it's RequireJS's responsibility to ensure that `config.paths` works with `define(name`, and that adding fixtures to React that assert that add noise more than value.

I feel more confident in this direction because I did test the old style before updating the fixture - they both work.

Reply:
Oh, I just saw that you did request both fixtures.  It got lost in the thread.  I'll add them.